src/lib/query.js

```diff
@@ -5,5 +5,10 @@ export default function(method, ...args) {
     return;
   }
 
-  window[options.globalObjectName](method, ...args);
-}
+  if (window[options.globalObjectName]) {
+    window[options.globalObjectName](method, ...args);
+  } else {
+    console.warn("gtag is not loaded yet")
```

Comment:
this should just never happen, in my opinion.
I need to add dataLayer and gtag immediately before loading the script, so that you can use it even when the script is not loaded yet